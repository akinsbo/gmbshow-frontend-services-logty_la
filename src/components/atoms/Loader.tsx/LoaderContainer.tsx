import * as React from 'react';
import Grid from '@material-ui/core/Grid';
import Loader from './Loader'

const LoaderContainer = (props: { loading: boolean }) =>
    <Grid container spacing={8}>
        <Grid item xs>
            <Loader loading={props.loading} />
        </Grid>
        <Grid item xs>
            <Loader loading={props.loading} />
        </Grid>
    </Grid>

export default  LoaderContainer