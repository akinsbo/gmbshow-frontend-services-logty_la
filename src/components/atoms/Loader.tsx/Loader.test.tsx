import * as React from "react"
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import LoaderContainer from './LoaderContainer'


describe('LoaderContainer', () => {
    it('should take a snapshot', () => {
        const { asFragment } = render(<LoaderContainer loading={true} />)
        expect(asFragment()).toMatchSnapshot()
    })
});



// test('handlers server error', async() => {
//     render(<LoaderContainer loading={true}>)
// await waitFor
// })