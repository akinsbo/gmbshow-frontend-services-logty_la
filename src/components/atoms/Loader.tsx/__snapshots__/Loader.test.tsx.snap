// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoaderContainer should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-8"
  >
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div>
        <div
          class="MuiTypography-root MuiTypography-body1"
        >
          <span
            class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse"
          />
        </div>
        <div
          class="MuiTypography-root MuiTypography-body1"
        >
          <span
            class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse"
          />
        </div>
        <div
          class="MuiTypography-root MuiTypography-body1"
        >
          <span
            class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse"
          />
        </div>
        <div
          class="MuiTypography-root MuiTypography-body1"
        >
          <span
            class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse"
          />
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div>
        <div
          class="MuiTypography-root MuiTypography-body1"
        >
          <span
            class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse"
          />
        </div>
        <div
          class="MuiTypography-root MuiTypography-body1"
        >
          <span
            class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse"
          />
        </div>
        <div
          class="MuiTypography-root MuiTypography-body1"
        >
          <span
            class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse"
          />
        </div>
        <div
          class="MuiTypography-root MuiTypography-body1"
        >
          <span
            class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
